pvlib/modelchain.py

```diff
@@ -1041,6 +1043,28 @@ def faiman_temp(self):
     def fuentes_temp(self):
         return self._set_celltemp(self.system.fuentes_celltemp)
 
+    @staticmethod
```

Comment:
We have a few helper functions at the bottom of the module. I wonder if this should live there rather than be a staticmethod on ModelChain. I don't have a strong preference at this time.

Reply:
For less experienced python users (like me) it would be more familiar as a helper function.

Reply:
Well, I had to lookup `staticmethod` in the python documentation to confirm its behavior before reviewing the function, count me in the less experienced group too.